distribution/zip/jballerina/bin/ballerina

```diff
@@ -164,8 +164,7 @@ if [ -n "${BAL_JAVA_DEBUG+set}" ]; then
 	if [ -n "$JAVA_OPTS" ]; then
 	  echo "Warning !!!. User specified JAVA_OPTS may interfere with BAL_JAVA_DEBUG"
 	fi
-	JAVA_OPTS="$JAVA_OPTS -Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=$BAL_JAVA_DEBUG"
-	echo "Please start the remote debugging client to continue..."
+	JAVA_OPTS="$JAVA_OPTS -Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=$BAL_JAVA_DEBUG,quiet=y"
```

Comment:
Shall we add a comment mentioning why we need to be running on quiet mode, to avoid breaking LS debug mode from future changes?

Reply:
Yes, good idea. will add.